uart_example/pubspec.yaml

```diff
@@ -0,0 +1,78 @@
+name: ble_testapp
+description: flutter ble testapp
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
```

Comment:
Please, clean-up the comments in this file. I understand they are generated automatically, but they are not useful (they can't replace the documentation, but cluttering the view).